index.js

```diff
@@ -150,6 +170,12 @@ class LedgerBridgeKeyring extends EventEmitter {
     }
     this.accounts = this.accounts.filter((a) => a.toLowerCase() !== address.toLowerCase())
     delete this.accountIndexes[ethUtil.toChecksumAddress(address)]
+    this._sendMessage({ action: 'ledger-close-bridge' }, () => this.forgetDevice())
+  }
+
+  updateTransportMethod(useLedgerLive = false) {
+    console.log("[LedgerBridgeKeyring] Sending message to iFrame to update Ledger Live preference to: ", useLedgerLive)
+    this._sendMessage({ action: 'ledger-update-transport', params: { useLedgerLive } })
```

Comment:
It would be better to pass a callback inhere so that the caller of `updateTransportMethod` knows when it has completed. Right now it's returning right away, possibly before the iframe has received the message.